src/Jellyfin.Database/Jellyfin.Database.Providers.SqLite/SqliteDatabaseProvider.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Jellyfin.Server.Implementations;
+using MediaBrowser.Common.Configuration;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Database.Providers.SqLite;
+
+/// <summary>
+/// Configures jellyfin to use an SQLite database.
+/// </summary>
+[JellyfinDatabaseProviderKey("Jellyfin-SQLite")]
```

Comment:
If the attribute is only used to expose the key value, you could also use a static property on the interface, which might also be less error-prone (like forgetting to add the attribute)

Reply:
nice idea but it seems that static abstract properties impose restrictions on that type beeing used as a generic which other parts of the code rely on which makes this unfeasable.
```
    /workspaces/jellyfin/Jellyfin.Server.Implementations/Extensions/ServiceCollectionExtensions.cs(24,91): error CS8920: The interface 'IJellyfinDatabaseProvider' cannot be used as type argument. Static member 'IJellyfinDatabaseProvider.ProviderKey.get' does not have a most specific implementation in the interface.
```

The only fix for that i can find is making the property virtual instead which defeats the purpose of implementing it on its own.